ui/ui-frontend/projects/vitamui-library/src/app/modules/object-editor/models/edit-object.model.ts

```diff
@@ -24,4 +24,8 @@ export interface EditObject extends DisplayObject {
   required?: boolean;
   virtual?: boolean;
   childrenChange: BehaviorSubject<EditObject[]>;
+  pattern?: string;
+  options?: string[];
+  hint?: string;
+  cardinality?: EffectiveCardinality;
```

Comment:
Non null (ou alors il faut refactor le code pour éviter les NPE) ?
```suggestion
  cardinality: EffectiveCardinality;
```

Reply:
`const cardinality = EffectiveCardinality || schemaElement.Cardinality;`

Ca n'arrive pas normalement. Mais je peux le mettre required.